docs/content/contribute/_index.md

```diff
@@ -37,9 +37,8 @@ Documentation https://ovh.github.io/cds is generated with Hugo. Source are under
 ```bash
 cd ${CDS_SOURCES}/docs/content/cli
 rm -rf cdsctl engine worker;
-cdsctl doc  # generate cdsctl directory documentation in the current directory so you must be inside ${CDS_SOURCES}/docs/content/cli
-engine doc  # generate engine directory documentation in the current directory so you must be inside ${CDS_SOURCES}/docs/content/cli
-worker doc  # generate worker directory documentation in the current directory so you must be inside ${CDS_SOURCES}/docs/content/cli
+cd ${CDS_SOURCES}
+GEN_PATH=${CDS_SOURCES}/docs/content/cli make doc 
```

Comment:
need go install worker, engine and cdsctl I think we must indicate this part in this example script